src/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+
+import { Person } from './types/Person';
+import { getPeople } from './api';
+import { Loader } from './components/Loader';
+import { NoPeople } from './NoPeople';
+import { LoadingError } from './components/LoadError';
+import { TablePeople } from './components/TablePeople';
+
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [newError, setNewError] = useState(false);
+  const [load, setLoad] = useState(true);
+
+  useEffect(() => {
+    getPeople()
+      .then((peopleFromServer) => {
+        setLoad(true);
+        setPeople(peopleFromServer);
+      })
+      .catch((error) => {
+        setNewError(true);
+        throw error;
+      })
+      .finally(() => {
+        setLoad(false);
+      });
+  }, []);
+
+  return (
+    <div className="block">
+      <h1 className="title">People Page</h1>
+      <div className="box table-container">
+        {load ? (
+          <Loader />
+        ) : (
+          <>
+            {newError && (
+              <LoadingError />
+            )}
+
+            {people.length === 0 && !load && (
```

Comment:
The '!load' condition is not required in the ternary operator since it's already guaranteed that 'load' is false in this context.